flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/CreditBasedPartitionRequestClientHandlerTest.java

```diff
@@ -430,23 +431,35 @@ private static void releaseResource(SingleInputGate inputGate, NetworkBufferPool
 	/**
 	 * Returns a deserialized buffer message as it would be received during runtime.
 	 */
-	private static BufferResponse createBufferResponse(
+	private BufferResponse createBufferResponse(
 			Buffer buffer,
 			int sequenceNumber,
-			InputChannelID receivingChannelId,
-			int backlog) throws IOException {
+			RemoteInputChannel receivingChannel,
```

Comment:
It is better to still use previous `receivingChannelId` instead, because we actually only need it.